src/backend/joanie/core/utils/webhooks.py

```diff
@@ -79,3 +81,57 @@ def synchronize_course_runs(serialized_course_runs):
                     webhook["url"],
                     extra=extra,
                 )
+
+
+def synchronize_signed_document(request):
+    """
+    Method to get the events from the signature provider.
+    We should return a 2xx status code after receiving the event
+    in order to be considered as successful.
+
+    If the event type (eventType) or the request method is not POST,
+    we shall raise a status code 4xx.
+
+    We will only update our objects in Joanie once the event type
+    has one of the following values only:
+    - ["workflowFinished", "recipientRefused"].
+    """
+    if request.method == "POST":
+        try:
+            event_data = json.loads(request.body)
```

Comment:
Only this line should be in the try:except. The rest of the code should come after the try/except